app/helpers/menu_helper.rb

```diff
@@ -96,6 +96,12 @@ def audience_link
     end
   end
 
+  def settings_link
+    can?(:manage, current_account)
```

Comment:
It seems you can't have a multiline ternary in ruby...
<img width="797" alt="Screenshot 2020-10-28 at 15 39 38" src="https://user-images.githubusercontent.com/13486237/97451310-eb801b80-1933-11eb-8763-26e7b593e7dc.png">

Reply:
D'oh 🤣